test/dash-playlist-loader.test.js

```diff
@@ -19,6 +19,7 @@ import {
 import '../src/videojs-http-streaming';
 import testDataManifests from 'create-test-data!manifests';
 import { sidx as sidxResponse } from 'create-test-data!segments';
+import {mp4VideoInit as mp4VideoInitSegment} from 'create-test-data!segments';
```

Comment:
Now we have to supply a legit init segment for sidx requests or else we blacklist the playlist. Had to update tests to do that.